packages/primitives/link/src/Root.native.tsx

```diff
@@ -21,10 +21,20 @@ export const Link = component(
       }
     },
   })
-)(({ children, id, onPress }) => (
-  <Text testID={id} onPress={onPress}>
-    {children}
-  </Text>
-))
+)(({ children, id, onPress }) => {
+  if (typeof children === 'string' || typeof children === 'number') {
```

Comment:
Is there any good reason the leave the part with `Text`?
Would it work or look differently if we would pass only a child containing a string to `TouchableWithoutFeedback`?

Reply:
AFAIK you cannot nest a Touchable within a Text, which is why we opted for a Text from the start. 

https://stackoverflow.com/a/52168520

This might have changed and I missed it, but I think we need to keep this being Text. We might want to add another prop that makes it "block level", but for that purpose there is the Button component instead.

Reply:
I can't find in that link that you can't nest Touchable withing Text 🤔 It says it looks misaligned, but maybe we can control that? And in the end we need something when it comes to links in paragraph with a small icon (for example indicating that the link goes to external website).

Reply:
@DonGideon did you check how it looks like in other project?
Maybe you could add meta file here and add it to sandbox in this repo?

Reply:
@irminakacz i guess we can remove the Text and only use touchable, but from looking online seems like Text is best practice (or most common practice) for text (like legal text) or for pagination (numbers)

@irminakacz and @xaviervia i dont seem to find the issue with it as well 🤔
but in our scenario it is a Text inside Tochable and not the other way around.
and in bubble-ui we use touchable for icons
and sure i can try some other usecases to see if something looks off